model/qti/datatype/BaseType.php

```diff
@@ -1,6 +1,6 @@
 <?php
 
-/*
+/**
```

Comment:
We will have to backport this to several places did we really need to refactor all those files?

Reply:
Okay, reverting